### PR TITLE
WWF-914 | Copy all git to docker

### PR DESCRIPTION
**Pull request by ShakedZrihen**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=46PtLXk)

### CODE REVIEW

**Review by orielz — APPROVED** [review]
Lgmt, I would consult with ofer about the descriptions, "copy user code to gitstream" sounds scary

File: action.yml [hunk 1]
```diff
@@ -33,32 +33,17 @@ runs:
...
       id: create-gitstream-folder
       shell: bash
       run: |
         mkdir gitstream
-    - name: Get Diff Base
-      id: get-diff
+    - name: copy usercode to gitstream folder
+      id: copy-user-code-to-gitstream-fodler
       shell: bash
       run: |
```
Comment (vim-zz) [inline action.yml]: Why do you need this step? i mean you could either set the path on the sep before to `gitstream/headbranch` 
Do you need 2 copies of this dir? you end up with 2 source dirs: `headbranch` and `gitstream/headbranch`

Comment (ShakedZrihen) [reply action.yml]: you are right, i don't need it

**Review by vim-zz — APPROVED** [review]
LGTM

**Pull request merged**